src/test/resources/unit/avrohugger-maven-plugin/mojo/overwrite/pom-overwrite.xml

```diff
@@ -53,6 +53,8 @@
                     </fileIncludes>
                     <typeOverrides>
                         <arrayType>SCALA_ARRAY</arrayType>
+                        <timestampMillisType>TIMESTAMP</timestampMillisType>
+                        <dateType>TIME_LOCALDATE</dateType>
```

Comment:
Could you change this two overrides to non-defaults and update the https://github.com/makubi/avrohugger-maven-plugin/blob/master/src/test/resources/unit/avrohugger-maven-plugin/mojo/TypeOverrides.avdl and https://github.com/makubi/avrohugger-maven-plugin/blob/master/src/test/resources/unit/avrohugger-maven-plugin/mojo/overwrite/TypeOverridesRecord.scala to include one field for each?